Source/Widgets/CesiumInspector/Cesium3DTilesInspectorViewModel.js

```diff
@@ -15,7 +15,7 @@ define([
         '../../Core/ScreenSpaceEventHandler',
         '../../Core/ScreenSpaceEventType',
         '../createCommand',
-        'ThirdParty/when'
+        '../../ThirdParty/when'
```

Comment:
For organizational purposes this include should now go before `../createCommand`.